_user/Makefile

```diff
@@ -0,0 +1,28 @@
+#
+# Makefile for user app
+#
+# Copyright 2021 Phoenix Systems
+#
+# %LICENSE%
+#
+
+include ../phoenix-rtos-build/Makefile.common
+
+CFLAGS += $(BOARD_CONFIG)
+
+.DEFAULT_GOAL := all
+
+DEFAULT_COMPONENTS := hello
+
+# default path for the programs to be installed in rootfs
+DEFAULT_INSTALL_PATH := /userbin
```

Comment:
why not `/usr/bin` ?

Reply:
We agreed on /usr/bin /usr/sbin. I don't know why.